pulsar-client-admin/src/main/resources/mltool/ManagedLedgerAdminTool.py

```diff
@@ -0,0 +1,261 @@
+        traceback.print_exc()
+        print 'No cursor found for {}/{}'.format(mlPath, cursorName)
+
+'''
+delete specific ledgerIds from the managed-ledger info
+Parameters
+----------
+zk : KazooClient
+   Zookeeper-client instance to query zk-client.
+mlPath : str
+    managed-ledger path
+deleteLedgerIds : str
+    comma separated deleting ledger-ids (eg: 123,124)
+eg:
+--zkServer localhost:2181 --mlPath sample/standalone/ns1/persistent/test --command delete-ml-ledgers --ledgerIds 3
+'''    
+def deleteMLLedgerIdsCommand(zk, mlPath, deleteLedgerIds):
+    try:
+        if deleteLedgerIds:
+            deletLedgerIds = set(deleteLedgerIds.split(","))
+            deletLedgerIdSet = set()
+            for id in deletLedgerIds:
+                deletLedgerIdSet.add(long(id))
+            print     deletLedgerIdSet
+            deleteLedgerIdsFromManagedLedgerInfo(zk, mlPath, deletLedgerIdSet)
+        else:
+            print 'Usage: --command {} [--ledgerIds]'.format(deleteMlLedgerIds)
+    except Exception as e:
+        traceback.print_exc()
+        print 'Failed to delete ml-ledger_ids {} due to {}'.format(mlPath, repr(e))
+
+'''
+Update mark-delete position of the given managed-cursor
+Parameters
+----------
+zk : KazooClient
+   Zookeeper-client instance to query zk-client.
+mlPath : str
+    managed-ledger path
+cursorName : str
+    managed-cursor path
+markDeletePosition: str
+    markDeletePosition combination of <ledgerId>:<entryId> (eg. 123:1)
+
+eg:
+--zkServer localhost:2181 --mlPath sample/standalone/ns1/persistent/test --cursorName s1 --cursorMarkDelete 0:1 --command update-mark-delete-cursor 
+'''    
+def updateMarkDeleteOfCursorCommand(zk, mlPath, cursorName, markDeletePosition):
+    try:
+        if cursorName:
+            if markDeletePosition:
+                positionPair = markDeletePosition.split(":")
+                if len(positionPair) == 2:
+                    updateCursorMarkDelete(zk, mlPath + "/" + cursorName, (long(positionPair[0])), long(positionPair[1]))
+                else:
+                    print "markDeletePosition must be in format <ledger_id>:<entry_id>"
+            else:
+                print 'Usage: --command {} [----cursorName] [--cursorMarkDelete]'.format(updateMakDeleteCursor)
+        else:
+            print 'Usage: --command {} [--cursorName] [--cursorMarkDelete]'.format(updateMakDeleteCursor)
+    except Exception as e:
+        traceback.print_exc()
+        print 'Failed to update ml-cursor {}/{} due to {}'.format(mlPath, cursorName, repr(e))
+            
+if __name__ in '__main__':
+        parser = argparse.ArgumentParser()
+        commandHelpText = 'Managed-ledger command: \n{}, {}, {}, {}'.format(printMlCommand, deleteMlLedgerIds, printCursorsCommands, updateMakDeleteCursor)
+        parser.add_argument("--zkServer", "-zk", required=True, help="ZooKeeperServer:port")
+        parser.add_argument("--command", "-cmd", required=True, help=commandHelpText)
```

Comment:
Instead of getting `--command` as an argument, to behave the same way as the other tools, can we get it as a positional parameter? eg: 

```
ManagedLedgerAdminTool.py print-cursor --zkServer localhost:2181 \
       --mlPath sample/standalone/ns1/persistent/test --cursorName s1
```